test/uk/gov/hmrc/uknwauthcheckerapi/controllers/AuthorisationsControllerSpec.scala

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmrc.uknwauthcheckerapi.controllers
+
+import org.apache.pekko.actor.setup.Setup
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.libs.json.Json
+import play.api.mvc.Result
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.uknwauthcheckerapi.models.{AuthorisationRequest, AuthorisationsResponse, AuthorisationResponse}
+import uk.gov.hmrc.uknwauthcheckerapi.generators.RandomGen.{eoriGen, localDateGen}
+
+import scala.concurrent.Future
+
+class AuthorisationsControllerSpec extends AnyWordSpec with Matchers {
+
+  private val controller = new AuthorisationsController(Helpers.stubControllerComponents())
+
+  "AuthorisationsController" should {
+
+    val date = localDateGen
+    val eori1 = eoriGen.sample.get
```

Comment:
Please use Generator Driven Property checks

See: https://www.scalatest.org/user_guide/generator_driven_property_checks

If you want to see how they can be brought in, see: 

https://github.com/hmrc/economic-crime-levy-registration/blob/8b5326b1a606cbdb0fca6b6c7d347a7a107f6ce3/test-common/uk/gov/hmrc/economiccrimelevyregistration/EclTestData.scala#L125

&

https://github.com/hmrc/economic-crime-levy-registration/blob/8b5326b1a606cbdb0fca6b6c7d347a7a107f6ce3/test/uk/gov/hmrc/economiccrimelevyregistration/connectors/IntegrationFrameworkConnectorSpec.scala#L84

You can do the same with the LocalDate Generator & the EORI generator.